src/cameraGeom/TransformMap.cc

```diff
@@ -454,7 +468,10 @@ class TransformMap::Factory : public table::io::PersistableFactory {
             connections.push_back(Connection{transform, fromSys, toSys});
         }
 
-        // deserialized connections should already be standardized
+        // Deserialized connections should already be standardized, but be
+        // defensive anyway.
+        auto const referenceSys = getReferenceSys(connections);
+        connections = standardizeConnections(referenceSys, std::move(connections));
```

Comment:
All the `std::move` are a bit unfortunate, I guess I imagined it would all fit on one line. :disappointed:

Do we need to worry about what the `referenceSys` is? I think I remember you saying something about how there's only one right answer...

Reply:
> All the std::move are a bit unfortunate

My turn to say, "that's C++[11] for you," I guess.  After playing around in Rust a bit (where move is actually the default, and that feels surprisingly natural), it's a real shame C++ didn't bite the bullet and add an operator for it - it's too fundamental to involve that many characters.

> Do we need to worry about what the referenceSys is? I think I remember you saying something about how PIXELS is the only right answer...

PIXELS is the only right answer for the `nativeSys` option in the Python configuration for `Detector`.  It's also true that the `TransformMap`s created by cameraGeom actually always use `FOCAL_PLANE` as the `referenceSys`, but `TransformMap` itself is better behaved than the classes that currently use it, and does not rely on that.